integration/setup/setup.go

```diff
@@ -322,3 +324,27 @@ func insertBenchmarkProvider(tb testtb.TB, ctx context.Context, collection *mong
 
 	return
 }
+
+// setupUser creates a user in admin database with PLAIN mechanism.
```

Comment:
Why do we create it with `PLAIN`? Is there any special logic related to `PLAIN`, or could it potentially be created with another mechanism too?

Reply:
Cool! Thank you :) It will help me in my task about sessions!